test/test_data/footer.html

```diff
@@ -0,0 +1,2 @@
+<img src="acme.png" alt="ACME Logo">
```

Comment:
This file is not present and the url should be "/img/acme.png". Probably we should add a integration test that checks for `400`'s.

Reply:
Well, it doesn't have to exist - I'm just using it as an example that you could add more than one DOM element. Could be a `<b>whatever</b>`, too; didn't want to pollute the repo with stuff that doesn't belong to kc-serve.

Reply:
Yeah, but you included a picture.jpg and not an acme.png